packages/editor-ui/src/mixins/expressionManager.ts

```diff
@@ -91,7 +91,7 @@ export const expressionManager = defineComponent({
 			fullTree.cursor().iterate((node) => {
 				const text = this.editor.state.sliceDoc(node.from, node.to);
 
-				if (skipSegments.includes(node.type.name)) return;
+				if (text === '' || skipSegments.includes(node.type.name)) return;
```

Comment:
I find this check very cryptic. What node are you checking for that has no text and why is it in the parsing output? Is it the error node? If so why is it erroring? Else could it be in skip segments?

Reply:
That's a good point. Turns out it's an error token inside the `CompositeIndetifier` I need to tweak the grammar a bit so support resolvables there also.